lib/cusparse/extra.jl

```diff
@@ -14,7 +14,7 @@ for (bname,gname,elty) in ((:cusparseScsrgeam2_bufferSizeExt, :cusparseScsrgeam2
     @eval begin
         function geam(alpha::Number, A::CuSparseMatrixCSR{$elty}, beta::Number, B::CuSparseMatrixCSR{$elty}, index::SparseChar)
             m, n = size(A)
-            (m, n) == size(B) && DimensionMismatch("dimensions must match: a has dims $(axes(A)), b has dims $(axes(B))")
+            (m, n) == size(B) || DimensionMismatch("dimensions must match: A has dims $(size(A)), B has dims $(size(B))")
```

Comment:
@maleadt 
Do you have an idea why it was working before. :thinking:

Reply:
Update: the `throw(...)` was missing.

Reply:
Haha, nice catch!